runtime/queries/purescript/highlights.scm

```diff
@@ -0,0 +1 @@
+; inherits: haskell
```

Comment:
You can copy this file to `runtime/queries/purescript/injections.scm` and `runtime/queries/purescript/locals.scm` to copy over the queries from those too